internal/tests/api/targets/classification_test.go

```diff
@@ -64,7 +64,7 @@ func TestTargets(t *testing.T) {
 					},
 					SessionMaxSeconds:           &wrapperspb.UInt32Value{Value: 0},
 					SessionConnectionLimit:      &wrapperspb.Int32Value{Value: 0},
-					WorkerFilter:                &wrapperspb.StringValue{Value: "worker-filter"},
+					EgressWorkerFilter:          &wrapperspb.StringValue{Value: "egress-worker-filter"},
```

Comment:
Do we actually want to remove WorkerFilter? Aren't we still returning that for a while?